pooch/core.py

```diff
@@ -374,12 +374,22 @@ def mydownloader(url, output_file, pooch):
 
         if action in ("download", "update"):
             action_word = dict(download="Downloading", update="Updating")
+
+            # Remove this if we switch to logging instead of warnings.
             warn(
                 "{} data file '{}' from remote data store '{}' to '{}'.".format(
                     action_word[action], fname, self.get_url(fname), str(self.path)
                 )
             )
 
+            get_logger().info(
+                "%s data file '%s' from remote data store '%s' to '%s'.",
+                action_word[action],
+                fname,
+                self.get_url(fname),
+                str(self.path),
```

Comment:
To make life easier, you can probably just keep the string formatting from the original warnings. I don't think `info` requires the arguments and would be just as happy to get a pure string as input. Then it's only a matter of replacing `warn` with `get_logger().info`.

Reply:
PyLint gives a warning if you invoke `format` yourself on the log message (see [this SO answer](https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation)), which is why I've written it the way I have. The supposed advantage of the way I've written it, and the reason that PyLint gives a warning if you don't, is that the string interpolation is lazy. If the user only wants to log critical errors, then you won't waste time formatting the strings at, say, the "info" level.

Personally, I don't like this argument -- why do string formatting differently in a few places for a likely negligible performance gain? I'd rather write it the way you suggest, but then we'd have to disable `logging-format-interpolation` in the project's PyLint configuration.

Reply:
Right, that makes sense if the app is logging quite a bit of information. Pooch doesn't log much but maybe it could be an issue in people want to download vast quantities of small files (probably not, but who knows). Might be best to just stick with what pylint suggests.